src/api/admin.ts

```diff
@@ -0,0 +1,61 @@
+import { DB, ServiceKey } from './types/admin'
+import { setData, getData, data } from './lib/data'
+import express from 'express'
+import fs from 'fs'
+
+const app = express.Router()
+
+const serviceKeys = getData('serviceKeys')
+
+app.post('/serviceHostsKey', (req, res) => {
+  // create service keys
+  const serviceHostKeys: ServiceKey = {
+    id: data.serviceKeys.length,
```

Comment:
I'm not sure if using the length is a good idea, because if a service key gets deleted, then the id could see a duplicate. For example:
4 keys are created. So the id will be: `0,1,2,3`

Then one key (`id: 1`) is deleted, so ids will be: `0,2,3`

If another key is created after that, then the ids will be `0,2,3,3`. This is a potential future bug. 

The I would generate ids via `uuid v4`: https://www.npmjs.com/package/uuid